RSA-SecureId/rsa-securedid/tests/test_parse_csv2.json

```diff
@@ -0,0 +1,14 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "rsa-securedId",
+        "dialect_uuid": "20876735-c423-4bbc-9d19-67edc91fb063"
+      }
+    },
+    "message": "11:26:43,377, example.intranet, audit.runtime.com.rsa.authmgr.internal.protocol.ace.AuthV4RequestHandler, ERROR, 6b746adf1d0646f7bcc518cd6ae4a16d,0e34d92f7c6549b19ed28471c02a049b,5.6.7.8,1.2.3.4,AUTH_PRINCIPAL_RESOLUTION,23008,FAIL,AUTH_RESOLUTION_FAILED_BY_ID_ALIAS,,,,,admin,,,09f1f5fc30e947ce9e564d5a91745091,000000000000000000001000e0011000,1.2.3.4,source.hostname,1,,,,,,,1,,,,,,,,"
+  },
+  "expected": {
+    "message": "11:26:43,377, example.intranet, audit.runtime.com.rsa.authmgr.internal.protocol.ace.AuthV4RequestHandler, ERROR, 6b746adf1d0646f7bcc518cd6ae4a16d,0e34d92f7c6549b19ed28471c02a049b,5.6.7.8,1.2.3.4,AUTH_PRINCIPAL_RESOLUTION,23008,FAIL,AUTH_RESOLUTION_FAILED_BY_ID_ALIAS,,,,,admin,,,09f1f5fc30e947ce9e564d5a91745091,000000000000000000001000e0011000,1.2.3.4,source.hostname,1,,,,,,,1,,,,,,,,"
```

Comment:
According to the tests with csv messages, the extraction of information doesn't work.